pkg/NETStandard.Library/NETStandard.Library.pkgproj

```diff
@@ -0,0 +1,21 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <PropertyGroup>
+    <Version>1.5.0</Version>
+    <SplitDependenciesBySupport>true</SplitDependenciesBySupport>
+  </PropertyGroup>
+
+  <Import Project="NETStandard.Library.packages.targets" />
+
+  <ItemGroup>
+    <ProjectReference Include="..\Microsoft.NETCore.Platforms\Microsoft.NETCore.Platforms.pkgproj">
+      <TargetFramework>netstandard1.0</TargetFramework>
+    </ProjectReference>
+    <ProjectReference Include="@(Package->'$(SourceDir)%(Identity)/pkg/%(Identity).pkgproj')" />
+    <!--ProjectReference Include="..\Microsoft.NETCore.Runtime\Microsoft.NETCore.Runtime.nuproj" /-->
```

Comment:
Any reason we shouldn't just remove the runtime dependency from here? I should have removed it when I removed the host dependency.